### ISSUE TITLE
Create a Pre-Built Image

### ISSUE THREAD

**Issue by conor-f** [catchouli/better_tactics#9]
Currently trying this out :)

Having the Docker image pre-built and hosted on a container registry would make it much quicker and easier to self-host/try it out. Here's an example from one of my repos where I tie it in with Github Actions/tagging if you are interested in something plug and play:

https://github.com/conor-f/fia_api/blob/main/.github/workflows/docker-hub-build-and-push.yml

**Comment by catchouli** [comment]
Nice, lemme know how it works for you :)

Where'd you find this, just out of curiosity? (I'm assuming most of the views are from the lichess discord right now.)

Good idea to set up CI/CD to build the image automatically and thanks for the example. I only just made the Dockerfile so I could host my own instance separately from my dev version and stop messing up my own database tbh 😁 but I'd use a prebuilt image myself too. (Although you can just use `docker-compose up --build` right now to make it easy and thinking about it I should add that to the readme too.)

**Comment by conor-f** [comment]
> Nice, lemme know how it works for you :)

It's pretty sweet! I wanted to make something like this for spaced repetition for openings instead of tactical puzzles but I hit a few stumbling blocks and moved on to other projects. This looks like it's off to a great start though! I'm going to throw it on my home server and check in on the repo every now and again to get new versions :)

Really I've got no feedback! It functions well, it's fast, straightforward, etc etc. Just needs some polish I guess but functionality wise it's all there for me :)

> Where'd you find this, just out of curiosity? (I'm assuming most of the views are from the lichess discord right now.)

I follow @ornicar on Github and saw that he gave the repo a star so I checked it out!
